middleware/validateBody.js

```diff
@@ -0,0 +1,10 @@
+require("express-async-errors");
```

Comment:
This middleware function is not currently utilized in the codebase. If it is not needed, please remove it to keep the code clean and maintainable.